test/clientWithSignupsDisabled.test.ts

```diff
@@ -21,3 +28,97 @@ test('signUp()', async () => {
   expect(data).toBeNull()
   expect(user).toBeNull()
 })
+
+test('generateLink() should be able to generate multiple links', async () => {
+  const invitedUser = faker.internet.email().toLowerCase()
+  const { error, data: firstInvite } = await authAdmin.generateLink('invite', invitedUser)
+  expect(error).toBeNull()
+  expect(firstInvite).toMatchInlineSnapshot(
+    {
+      id: expect.any(String),
+      confirmation_sent_at: expect.any(String),
+      email: expect.any(String),
+      phone: expect.any(String),
+      created_at: expect.any(String),
+      aud: expect.any(String),
+      updated_at: expect.any(String),
+      invited_at: expect.any(String),
+      app_metadata: {
+        provider: 'email',
+      },
+    },
+    `
+    Object {
+      "action_link": "",
```

Comment:
@kangmingtay - it seems like the invite link isn't generated if signups are disabled?